vernac/g_vernac.mlg

```diff
@@ -113,7 +113,8 @@ GRAMMAR EXTEND Gram
     ]
   ;
   attribute:
-    [ [ k = ident ; v = attribute_value -> { Names.Id.to_string k, v } ]
+    [ [ k = ident ; v = attribute_value -> { Names.Id.to_string k, v }
+      | "using" ; v = attribute_value -> { "using", v } ]
```

Comment:
Can parser experts make a general `ident_or_keyword` entry instead of hardcoding it here?
@coq/parsing-maintainers

Reply:
that would be nice indeed

Reply:
Regarding declaration of tokens, a direction I'm currently exploring is rather to have locally defined keywords, that is, each rule (or rule prefixes) declares the keywords specific to its.